### DISCUSSION TITLE
Camera installation on Fedora 36

### DISCUSSION THREAD

**Discussion by FlyingPC** [linux-surface/linux-surface#824] [Q&A]
Hi,

I'm trying to use cameras on my Surface Go 2.

I think I have followed every steps describe in this page : https://github.com/linux-surface/linux-surface/wiki/Camera-Support but it doesn't work.

In few words, I have installed surface kernel and boot with it. Then, I have build and install libcamera from git repository (with no error).

But when I try : cam --list there is no camera listed.

And dmesg | grep ipu3 return :
"""
[    3.965646] ipu3_imgu: module is from the staging directory, the quality is unknown, you have been warned.
[    3.967644] ipu3-imgu 0000:00:05.0: device 0x1919 (rev: 0x1)
[    3.967683] ipu3-imgu 0000:00:05.0: physical base address 0x00000000b1000000, 4194304 bytes
[    4.110532] ipu3-imgu 0000:00:05.0: loaded firmware version irci_irci_ecr-master_20161208_0213_20170112_1500, 17 binaries, 1212984 bytes
"""
I think maybe i didn't installed correctly ipu3-cio2 but I don't know where is it possible to find it in fedora repository.

I open this discussion to obtain some help.

Thanks in advance for any help.

Regards.

FPC

**Comment by NP-chaonay** [comment]
> module is from the staging directory, the quality is unknown, you have been warned.

I think this warning when build the module from dev/beta branch (idk if it true). so the actual problem msg may not be there.
